jenkins/marvell-armhf/buildimage-mrvl-armhf-all/Jenkinsfile

```diff
@@ -39,10 +39,19 @@ pipeline {
 git submodule foreach --recursive '[ -f .git ] && echo "gitdir: $(realpath --relative-to=. $(cut -d" " -f2 .git))" > .git'
 export DOCKER_DATA_ROOT_FOR_MULTIARCH=/data/march/docker
 make configure PLATFORM=marvell-armhf PLATFORM_ARCH=armhf
-make SONIC_BUILD_JOBS=4 INSTALL_DEBUG_TOOLS=y target/sonic-marvell-armhf.bin || make SONIC_BUILD_JOBS=2 INSTALL_DEBUG_TOOLS=y target/sonic-marvell-armhf.bin || make SONIC_BUILD_JOBS=1 INSTALL_DEBUG_TOOLS=y target/sonic-marvell-armhf.bin
-mv target/sonic-marvell-armhf.bin target/sonic-marvell-armhf-dbg.bin
+#
+# Skip debug build for now until build time improvements can be made
+# make SONIC_BUILD_JOBS=4 INSTALL_DEBUG_TOOLS=y target/sonic-marvell-armhf.bin || make SONIC_BUILD_JOBS=2 INSTALL_DEBUG_TOOLS=y target/sonic-marvell-armhf.bin || make SONIC_BUILD_JOBS=1 INSTALL_DEBUG_TOOLS=y target/sonic-marvell-armhf.bin
+# mv target/sonic-marvell-armhf.bin target/sonic-marvell-armhf-dbg.bin
+#
+# Build as much as we can concurrently, backing off if failure
 make SONIC_BUILD_JOBS=4 target/sonic-marvell-armhf.bin || make SONIC_BUILD_JOBS=2 target/sonic-marvell-armhf.bin || make SONIC_BUILD_JOBS=1 target/sonic-marvell-armhf.bin
-sudo docker -H unix:///var/run/march/docker.sock system prune -f
+#
+# Cleanup for next build (dockerd and docker multi-arch artifacts)
+if sudo [ -f dockerfs/var/run/docker.pid ] ; then
+    pid=`sudo cat dockerfs/var/run/docker.pid` ; sudo kill $pid
```

Comment:
we have arm64 build as well, will this disrupt arm64 build if it is in progress?

Reply:
The above logic is executed in the context of the current armhf build and thus will not impact an in-flight arm64 build. Each arm build instance gets its own instance of dockerd to perform the build.